CHANGELOG.md

```diff
@@ -33,6 +33,9 @@ _None_
 * Fonts/IB/JSON/Plist/Strings/XCAssets: all templates that load data at runtime from a bundle now support a `bundle` template parameter, which you can use to override the bundle from which resources are loaded. Check out the [template specific documentation](Documentation/templates/) for more information.  
   [David Jennes](https://github.com/djbe)
   [#737](https://github.com/SwiftGen/SwiftGen/pull/737)
+* Fonts/IB/JSON/Plist: Similar to the `strings` templates, these templates now support a `lookupFunction` template parameter, which allows you to provide your own resource lookup mechanism at runtime. Check the [template specific documentation](Documentation/templates/) for more information.  
+  [David Jennes](https://github.com/djbe)
+  [#738](https://github.com/SwiftGen/SwiftGen/pull/738)
```

Comment:
Unrelated to that PR, but since we're on spelling consistency on line 29 I'd change s/XCassets/XCAssets/ in that CHANGELOG, since we use the `XCAssets` capitalization everywhere else